x-pack/plugins/upgrade_assistant/public/application/components/overview/fix_logs_step/fix_logs_step.tsx

```diff
@@ -34,6 +36,25 @@ const i18nTexts = {
       defaultMessage: 'Resolve deprecation issues and verify your changes',
     }
   ),
+  apiCompatibilityNoteTitle: i18n.translate(
+    'xpack.upgradeAssistant.overview.apiCompatibilityNoteTitle',
+    {
+      defaultMessage: 'Apply API compatibility headers (optional)',
+    }
+  ),
+  apiCompatibilityNoteBody: (docLink: string) => (
+    <FormattedMessage
+      id="xpack.upgradeAssistant.overview.apiCompatibilityNoteBody"
+      defaultMessage="We recommend you resolve all deprecation issues before upgrading. However, it can be challenging to ensure all requests are fixed. For additional safety, include API version compatibility headers in your requests. {learnMoreLink}."
+      values={{
+        learnMoreLink: (
+          <EuiLink href={docLink} target="_blank">
+            Learn more
```

Comment:
This needs to be translated too

Reply:
Oh thanks for spotting that! I pushed a small fix to address it https://github.com/elastic/kibana/pull/110469/commits/0bc20f121b14d55db97c0538fab430a4e86e8311